styleguide/components/21-atoms/inputs/input-date/_input-date.scss

```diff
@@ -1,157 +1,7 @@
-  background-size: 1.25rem;
+  // background: $field-background url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPCFET0NUWVBFIHN2ZyBQVUJMSUMgIi0vL1czQy8vRFREIFNWRyAxLjEvL0VOIiAiaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkIj4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSIxMDI0IiBoZWlnaHQ9IjEwMjQiIHZpZXdCb3g9IjAgMCAxMDI0IDEwMjQiPjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTk1Ny42MDIgMTcyLjczN3Y3NTIuNDAyaC04ODkuMjAzdjY4LjQwMmg5NTcuNjAydi04MjAuODAzek04ODkuMjAzIDEwNC4zMzhoLTEwMi42YzAgNTguMTQyLTQ0LjQ2MSAxMDIuNi0xMDIuNiAxMDIuNnMtMTAyLjYtNDQuNDU5LTEwMi42LTEwMi42aC0yMzkuNGMwIDU4LjE0Mi00NC40NTkgMTAyLjYtMTAyLjYgMTAyLjZzLTEwMi42LTQ0LjQ1OS0xMDIuNi0xMDIuNmgtMTM2Ljc5OXY3NTIuNDAyaDg4OS4yMDN2LTc1Mi40MDJ6TTgyMC44MDMgNzg4LjM0aC03NTIuNDAydi01MTMuMDAyaDc1Mi40MDJ2NTEzLjAwMnpNMjM5LjQgMTM4LjUzN2MyMC41MiAwIDM0LjE5OS0xMy42ODEgMzQuMTk5LTM0LjIwMXYtNjguNDAyYzAtMjAuNTItMTMuNjgxLTM0LjE5OS0zNC4yMDEtMzQuMTk5cy0zNC4yMDEgMTMuNjc5LTM0LjIwMSAzNC4xOTl2NjguNDAyYzAgMjAuNTIgMTMuNjc5IDM0LjIwMSAzNC4yMDEgMzQuMjAxek02ODQuMDA0IDEzOC41MzdjMjAuNTIgMCAzNC4yMDEtMTMuNjgxIDM0LjIwMS0zNC4yMDF2LTY4LjQwMmMwLTIwLjUyLTEzLjY4MS0zNC4xOTktMzQuMjAxLTM0LjE5OXMtMzQuMjAxIDEzLjY3OS0zNC4yMDEgMzQuMTk5djY4LjQwMmMwIDIwLjUyIDEzLjY4MSAzNC4yMDEgMzQuMjAxIDM0LjIwMXpNMjA1LjIwMSAzNzcuOTM4aDEwMi42djEwMi42aC0xMDIuNnYtMTAyLjZ6TTQxMC40MDIgMzc3LjkzOGgxMDIuNnYxMDIuNmgtMTAyLjZ2LTEwMi42ek01ODEuNDA0IDM3Ny45MzhoMTAyLjZ2MTAyLjZoLTEwMi42di0xMDIuNnpNMjA1LjIwMSA1NDguOTM4aDEwMi42djEwMi42aC0xMDIuNnYtMTAyLjZ6TTQxMC40MDIgNTQ4LjkzOGgxMDIuNnYxMDIuNmgtMTAyLjZ2LTEwMi42ek01ODEuNDA0IDU0OC45MzhoMTAyLjZ2MTAyLjZoLTEwMi42di0xMDIuNnoiIGZpbGw9InJnYig2OCwgNjgsIDY4KSI+PC9wYXRoPgo8L3N2Zz4K') no-repeat right .625rem center;
+  // background-size: 1.25rem;
```

Comment:
Remove comments

Reply:
Will do so when the designs of all form atoms are final.